PRIVACY_POLICY.zh-CN.md

```
@@ -67,10 +67,11 @@ Hackers' Pub 是一个面向软件工程师和技术爱好者的邀请制社交
 您添加到主页的外部服务链接（如 GitHub、Mastodon 或个人网站），及其加密验证
 状态。
 
-### 推送通知（iOS）
+### 推送通知
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)